types/http-proxy-middleware/index.d.ts

```diff
@@ -15,7 +15,7 @@ import * as httpProxy from "http-proxy";
 import { Readable } from 'stream';
 
 declare function proxy(config: proxy.Config): proxy.Proxy;
-declare function proxy(contextOrUri: string | string[] | proxy.Filter, config: proxy.Config): proxy.Proxy;
+declare function proxy(contextOrUri: string | string[] | proxy.Filter, config?: proxy.Config): proxy.Proxy;
 
 declare namespace proxy {
   type Filter = (pathanme: string, req: http.IncomingMessage) => boolean;
```

Comment:
I know you didn't change this, but yould you please fix the typo in `pathanme`?

Reply:
Yep, done !

Reply:
Thank you!